raiden/api/rest.py

```diff
@@ -1353,10 +1361,10 @@ def patch_channel(
                 errors="Requested channel for token {} and partner {} not found".format(
                     to_checksum_address(token_address), to_checksum_address(partner_address)
                 ),
-                status_code=HTTPStatus.CONFLICT,
+                status_code=HTTPStatus.NOT_FOUND,
```

Comment:
a change of error codes must be reflected in the API documentation.
Actually this whole PR must be documented there: https://github.com/raiden-network/raiden/blob/develop/docs/rest_api.rst#channel-management